src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -664,11 +677,13 @@ class BinaryCompactObject : public DomainCreator<3> {
       std::array<double, 2> initial_size_map_accelerations,
       std::array<std::string, 2> size_map_function_of_time_names,
       Object object_A, Object object_B, double radius_enveloping_cube,
-      double radius_enveloping_sphere,
+      double outer_radius,
```

Comment:
I think I would change this to `outer_radius_domain` or `domain_radius`, also so that `outer_radius` is not confused with names like `outer_radius_objectA`